src/impala/emit.cpp

```diff
@@ -31,15 +31,15 @@ class CodeGen : public IRBuilder {
         return Def();
     }
 
```

Comment:
I changed this in 3cbffd6 so that there is a common API for emit_jump(). Before, the call to emit_jump was ambiguous.